dashboards/Edge-LB/Edge-LB-Cloud-Controller.json

```diff
@@ -0,0 +1,1152 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 0,
+  "id": 52,
```

Comment:
this needs to be `null` - the `Export for sharing externally` option needs to be checked when exporting! 
![image](https://user-images.githubusercontent.com/5897740/63535380-2aac1480-c4c6-11e9-9f61-e3393421c430.png)